src/renderer/contexts/GlobalNodeState.tsx

```diff
@@ -551,12 +551,13 @@ export const GlobalProvider = memo(
         );
 
         const createNode = useCallback(
-            (proto: NodeProto): void => {
+            (proto: NodeProto, parent?: Node<NodeData>): void => {
```

Comment:
I think it would be better if this was a node id. This would probably play together nicer with `hoveredNode` since you could do something like `const parentId = hoveredNode ?? parentIdParameter` and then search for that.